app/views/super_admin/notifications/_form.html.erb

```diff
@@ -28,6 +28,13 @@
     </div>
   </div>
 
+  <div class="row">
+    <div class="form-group col-xs-12">
+      <%= f.check_box :active, style: 'width: auto' %>
```

Comment:
don't use style. Use a class instead. reducing `col-xs-12` down to 8 or 6 or whatever should do the trick